underlay/src/main/resources/config/indexer/emerge/emerge20230831.json

```diff
@@ -0,0 +1,25 @@
+{
+  "underlay": "emerge",
```

Comment:
Right, this value needs to point to an [underlay directory](https://github.com/DataBiosphere/tanagra/tree/main/underlay/src/main/resources/config/underlay) that contains 2 files: `underlay.json` and `ui.json`. You could copy from one of the existing underlays as a starting point (e.g. copy the contents of `sd`).